static/sentry.ts

```diff
@@ -1,4 +1,3 @@
-// Copyright (c) 2023, Compiler Explorer Authors
```

Comment:
We should probably reinstate this

Reply:
I think there's no need - the year is that of publishing 
https://www.copyrightlaws.com/copyright-symbol-notice-year/

Reply:
I meant...you deleted this line. It was there before and was OK, I was going to put it back as it's part of the rest of the codebase. Or am I missing something?

Reply:
Oh, you're right of course. Sorry.